### PR TITLE
Update readmes

### PR DESCRIPTION
**Pull request by assoomro**

### CODE REVIEW

File: docs/run-azure.md [hunk 1]
```diff
@@ -50,8 +49,12 @@ The deployment is also hooked up to the PR process, which will create another en
...
-- `tear-down-azure.sh <namespace>` to remove the deployment
-- Remove any created DBs in the azure database server
-- Remove the created container on the azure storage account
\ No newline at end of file
+*****NOTE*****
+In order for the script to also clear the databases and storage containers, you'll need to be connected to the VPN
+
+Run the tear down script 
+
+`tear-down-azure.sh <namespace> -a <blob store account connection string>`
```
Comment (talawson05) [inline docs/run-azure.md:58]: shouldn't the connection string be in quotes?

Comment (talawson05) [reply docs/run-azure.md:58]: do you not need -n for namespace?

Comment (assoomro) [reply docs/run-azure.md:58]: You do indeed, good point in both cases, thanks!

**Review by talawson05 — APPROVED** [review]
💯

**Review by gavinwakeley1 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**